staging/src/k8s.io/apiserver/pkg/util/feature/feature_gate.go

```diff
@@ -176,6 +184,7 @@ func (f *featureGate) Add(features map[Feature]FeatureSpec) error {
 	return nil
 }
 
+// Enabled returns true if the key is enabled.
 func (f *featureGate) Enabled(key Feature) bool {
```

Comment:
These functions are not generated in the docs because `featureGate` is private. Is there any harm in making it exported?